packages/contracts/test/voting/token-voting-setup.ts

```diff
@@ -25,18 +26,19 @@ const prepareInstallDataTypes = [
   'tuple(address[],uint256[])',
 ];
 
+const participationThreshold = 25;
+const supportThreshold = 50;
+const minDuration = 10;
+
 // minimum bytes for `prepareInstallation` data param.
 const MINIMUM_DATA = abiCoder.encode(prepareInstallDataTypes, [
-  1,
-  1,
-  1,
+  participationThreshold,
+  supportThreshold,
+  minDuration,
```

Comment:
Is `minDuration` expressed in blocks? seconds?

Reply:
All dates and durations are expressed as timestamps and timespans in seconds.